software/index.html

```diff
@@ -36,96 +37,127 @@
-                            <a class="list-group-item" href="/software/mentor/">Join as a Mentor</a>
-                            <a class="list-group-item" href="/software/history/">Past Sessions</a>
-                        </ul>
+                 <!-- Main Logo -->
+                <div class="text-center">
+                    <img src="./images/logo_white.png" class="pl-3" alt="Software Saturdays Logo" width="350" >
+                    <h1 class="display-1 fw-bold text-light">Software Saturdays</h1>
+                    <div class="container-fluid">
+                        <img src="./images/IEEE_white.png" alt="IEEE Logo" width="200">
+                        <img src="./images/coe_white.png" alt="College of Engineering Logo" width="200">
                     </div>
-                    <div class="col-md-9 text-center">
-                        <img src="./images/logo_white.jpg" alt="Software Saturdays Logo" width="300">
-                        <h1 class="display-3 fw-bold">Software Saturdays</h1>
-                        <p class="fs-4">Sponsored by the Purdue College of Engineering</p>
+                     <!-- Registration Buttons -->
+                    <div class="container-fluid d-flex flex-row justify-content-center text-center">
+                        <a href="https://forms.gle/C5XQDh6ATyFnwfdX7" target="_blank" type="button" class="btn bg-purdue btn-lg px-4 mx-2">S23 Student Registration</a>
+                        <a href="https://forms.gle/PNS4h3NUN37c7X7D7" target="_blank" type="button" class="btn btn-secondary btn-lg px-4 mx-2">S23 Mentor Application</a>
                     </div>
                 </div>
             </div>
 
-            <div class="my-2 container-fluid bg-pieee">
-                <div class="container-lg p-5">
-
-                    <h2 class="fs-1">Teaching modern software tools to the modern Purdue engineer.</h2>
-                    <p class="fs-5 text-start">
-                        Purdue IEEE is partnering with the Purdue College of Engineering to offer Software Saturdays, a FREE course available to all Purdue students!
-                    </p>
-                    <p class="fs-5 text-start">
-                        Software Saturdays introduces software development and programming frameworks to our attendees through a project-based learning experience.
-                        We introduce concepts and projects that are useful regardless of major: in the 21st century, having basic programming knowledge is vital no matter your hobbies or jobs you pursue.
-                    </p>
-                    <p class="fs-5 text-start">
-                        Sponsored by the Purdue College of Engineering, we are able to offer an official Purdue College of Engineering certification for everyone who attends!
+             <!-- Basic Info -->
+            <div class="my-3 p-5 container-fluid fade-in-left">
```

Comment:
Wide text does not look good on a large screen, it's hard to scan back and forth. Try a responsive container.